src/js/controller/model.js

```diff
@@ -260,7 +260,12 @@ define([
             return this[attr];
         },
         'set' : function(attr, val) {
+            if (this[attr] === val) {
+                return;
+            }
             this[attr] = val;
+            this.trigger('change:'+attr, this, val);
+            this.trigger('change', this);
```

Comment:
True, I'll just document our discussion for the rest of the team.

My intention was to mimic the API of backbones implementation of Model, however we agreed that it wasn't the time to change our internal strategy, especially not if we would do so only partially.